tasks/clean-references.mjs

```diff
@@ -1,8 +1,8 @@
-import references from '../src/references/raw-references.json' assert { type: 'json' };
```

Comment:
@calebpollman The original process proposed have the JS repo generate and check in this raw file, which the docs repo would then transform into the `apiReferences.json` file, which I have moved to scop by package name.

My update process proposes that the JS repo (and other repos) should generate their api docs json files, then use the docs repo transformation logic proposed here to transform the file directly into the finished product instead of having multiple interstitial json files checked in along the way.

I had originally included updating the amplify-js file with this change, but it seems better to do this using the automated process and leave this change a no-op for the docs.